client/oci.go

```diff
@@ -585,9 +595,9 @@ func (e *unexpectedArchitectureError) Is(target error) bool {
 
 var errDigestNotVerified = errors.New("digest not verified")
 
-func (r *ociRegistry) getImageConfig(ctx context.Context, c credentials, name string, d digest.Digest) (imageConfig, error) {
+func (r *ociRegistry) getimageConfig(ctx context.Context, creds credentials, name string, d digest.Digest) (imageConfig, error) {
```

Comment:
Case typo here I think?

```suggestion
func (r *ociRegistry) getImageConfig(ctx context.Context, creds credentials, name string, d digest.Digest) (imageConfig, error) {
```